appliances/Community/Lithops/appliance.sh

```diff
@@ -0,0 +1,255 @@
+        exit 1
+    fi
+
+    msg info "Install pip dependencies"
+    if ! pip install "${2}" ; then
+        msg error "Python pip dependencies installation failed"
+        exit 1
+    fi
+}
+
+install_docker()
+{
+    msg info "Add Docker official GPG key"
+    install -m 0755 -d /etc/apt/keyrings
+
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+
+    chmod a+r /etc/apt/keyrings/docker.asc
+
+    msg info "Add Docker repository to apt sources"
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+        $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+        sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    apt update
+
+    msg info "Install Docker Engine"
+    if ! apt-get install -y docker-ce=$DOCKER_VERSION docker-ce-cli=$DOCKER_VERSION containerd.io docker-buildx-plugin docker-compose-plugin ; then
+        msg error "Docker installation failed"
+        exit 1
+    fi
+}
+
+install_lithops()
+{
+    msg info "Install Lithops from pip"
+    if ! pip install lithops==${LITHOPS_VERSION} ; then
+        msg error "Error installing Lithops"
+        exit 1
+    fi
+
+    msg info "Create /etc/lithops folder"
+    mkdir /etc/lithops
+
+    return $?
+}
+
+create_lithops_config()
+{
+    msg info "Create default config file"
+    cat > /etc/lithops/config <<EOF
+lithops:
+  backend: localhost
+  storage: localhost
+
+# Start Compute Backend configuration
+# End Compute Backend configuration
+
+# Start Storage Backend configuration
+# End Storage Backend configuration
+EOF
+}
+
+update_lithops_config(){
+    msg info "Update compute and storage backend modes"
+    sed -i "s/backend: .*/backend: ${ONEAPP_BACKEND}/g" /etc/lithops/config
+    sed -i "s/storage: .*/storage: ${ONEAPP_STORAGE}/g" /etc/lithops/config
+
+    if [ ${ONEAPP_STORAGE} = "localhost" ]; then
+        msg info "Edit config file for localhost Storage Backend"
+        sed -i -ne "/# Start Storage/ {p;" -e ":a; n; /# End Storage/ {p; b}; ba}; p" /etc/lithops/config
+    elif [ ${ONEAPP_STORAGE} = "minio" ]; then
+        msg info "Edit config file for MinIO Storage Backend"
+        if ! check_minio_attrs; then
+            echo
+            msg error "MinIO configuration failed"
+            msg info "You have to provide endpoint, access key id and secrec access key to configure MinIO storage backend"
+            exit 1
+        else
+            msg info "Adding MinIO configuration to /etc/lithops/config"
+            sed -i -ne "/# Start Storage/ {p; iminio:\n  endpoint: ${ONEAPP_MINIO_ENDPOINT}\n  access_key_id: ${ONEAPP_MINIO_ACCESS_KEY_ID}\n  secret_access_key: ${ONEAPP_MINIO_SECRET_ACCESS_KEY}\n  storage_bucket: ${ONEAPP_MINIO_BUCKETT}" -e ":a; n; /# End Storage/ {p; b}; ba}; p" /etc/lithops/config
+        fi
+    fi
+}
+
+check_minio_attrs()
+{
+    [ -z "$ONEAPP_MINIO_ENDPOINT" ] && return 1
```

Comment:
For consistency:

```suggestion
    [[ -z "$ONEAPP_MINIO_ENDPOINT" ]] && return 1
```

also in other lines in this function.